### PR TITLE
Improve session.close() analysability

### PR DESCRIPTION
**Pull request by gvanbrakel**

### CODE REVIEW

File: cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisHttpSender.java [hunk 1]
```diff
@@ -150,7 +150,7 @@ public Message extractResult(HttpResponseHandler responseHandler, PipeLineSessio
 			responseCode = statusline.getStatusCode();
 
 			Message responseMessage = responseHandler.getResponseMessage();
-			responseMessage.closeOnCloseOf(session);
+			responseMessage.closeOnCloseOf(session, this);
```
Comment (nielsm5) [inline cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisHttpSender.java:153]: HTTP connection of CMIS call

File: core/src/main/java/nl/nn/adapterframework/http/HttpSender.java [hunk 2]
```diff
@@ -553,7 +553,7 @@ protected Message extractResult(HttpResponseHandler responseHandler, PipeLineSes
 		if (response==null) {
 			Message responseMessage = responseHandler.getResponseMessage();
 			if(!Message.isEmpty(responseMessage)) {
-				responseMessage.closeOnCloseOf(session);
+				responseMessage.closeOnCloseOf(session, this);
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/http/HttpSender.java:556]: connection of HTTP call [xyz]

File: core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java [hunk 3]
```diff
@@ -167,7 +167,7 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 			}
 		}
 
-		message.closeOnCloseOf(session); // avoid connection leaking when the message itself is not consumed.
+		message.closeOnCloseOf(session, this); // avoid connection leaking when the message itself is not consumed.
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java:170]: Pleister! This should be registered as closeOnCloseOf on the cause, not the effect!

**Review by nielsm5 — CHANGES_REQUESTED** [review]

File: core/src/main/java/nl/nn/adapterframework/senders/SenderWrapperBase.java [hunk 4]
```diff
@@ -96,7 +95,7 @@ public Message sendMessage(Message message, PipeLineSession session) throws Send
 
 	@Override
 	public String getLogPrefix() {
-		return ClassUtils.nameOf(this)+" ["+getName()+"] ";
+		return super.getLogPrefix();
 	}
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/senders/SenderWrapperBase.java:99]: Dan kan dit weg toch

**Review by nielsm5 — CHANGES_REQUESTED** [review]

File: core/src/main/java/nl/nn/adapterframework/senders/SenderWrapperBase.java (hunk 4 above)
Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/senders/SenderWrapperBase.java:99]: dat dacht ik ook, maar super is protected

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**